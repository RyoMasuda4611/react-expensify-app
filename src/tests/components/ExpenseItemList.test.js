import React from 'react';
import { shallow } from 'enzyme';
import ExpenseItemList from '../../components/ExpenseItemList';
import expenses from '../fixtures/expenses';

test('should render Expense Item List correctly', () => {
    const wrapper = shallow(<ExpenseItemList {...expenses[0]} />);
    expect(wrapper).toMatchSnapshot();
});
