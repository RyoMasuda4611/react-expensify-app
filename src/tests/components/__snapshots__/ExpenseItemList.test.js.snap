// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense Item List correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    10
    -
    0
  </p>
</div>
`;
